docs/src/documentation/01-getting-started/04-support/03-team.mdx

```diff
@@ -5,23 +5,10 @@ redirect_from:
 ---
 
 import GithubContributors from "../../../components/Github";
-import OrbitTeam from "../../../components/Github/OrbitTeam";
 
 We're continuously improving and supporting our React components,
 UI kit for Figma, and the documentation on orbit.kiwi.
 Occasionally, we also introduce some small tooling
 to support better workflows for designers and developers.
 
-<OrbitTeam />
-
-## All contributors
-
-From small reviews or additions to contributions of whole components.
-We really appreciate all your efforts when helping us to make Orbit better
-👏
-
-### React components
-
 <GithubContributors />
-
-Thanks to all these folks for caring, your contribution helps to all Orbit consumers.
```

Comment:
I would like to avoid moving MDX to JSX as much as possible. Is there a way to get what you wanted, but also keep this content in MDX?

Reply:
I thought for this particular case there is no need to have it inside `MDX`, it was in `MDX` because it had two separate components for `OrbitTeam` and `Contributors`. Now it's one component because it's fetching all data from graphql, not like before from json and github.